### PR TITLE
Add tuner parameter in cparams

### PR DESCRIPTION
**Pull request by martaiborra**
With this PR now you can specify the tuner to use like you would do with the codec.

### CODE REVIEW

File: blosc2/core.py [hunk 1]
```diff
@@ -1230,7 +1230,11 @@ def compute_chunks_blocks(shape, chunks=None, blocks=None, dtype=np.uint8, **kwa
                     cparams2["filters"][i] = blosc2.Filter.NOFILTER
         else:
             cparams2 = cparams
+        # Do not apply specific tuner
+        aux_tuner = cparams2["tuner"] if "tuner" in cparams2 else blosc2.Tuner.STUNE
+        cparams2["tuner"] = blosc2.Tuner.STUNE
```
Comment (FrancescAlted) [inline blosc2/core.py:1235]: Can you add a comment above like "# Force STUNE to get a hint on the blocksize"

File: blosc2/core.py [hunk 2]
```diff
@@ -1230,7 +1230,11 @@ def compute_chunks_blocks(shape, chunks=None, blocks=None, dtype=np.uint8, **kwa
...
         src = blosc2.compress2(np.zeros(nitems, dtype=f"V{itemsize}"), **cparams2)
+        cparams2["tuner"] = aux_tuner
```
Comment (FrancescAlted) [inline blosc2/core.py:1237]: I think moving this line one position below would increase readability

File: doc/reference/autofiles/top_level/blosc2.Tuner.rst [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+.. currentmodule:: blosc2
+
+.. autoclass:: Tuner
+   
+   .. rubric:: Attributes
+
+   .. autosummary::
+   
+      ~Tuner.STUNE
+      ~Tuner.BTUNE
```
Comment (FrancescAlted) [inline doc/reference/autofiles/top_level/blosc2.Tuner.rst:13]: I'd add some docstrings here:

STUNE: A 'simple' tuner.  This is the default in the Blosc2 library

BTUNE: A more sophisticated tuner that can select different codecs/filters for different chunks.  More info: https://btune.blosc.org

**Review by FrancescAlted — CHANGES_REQUESTED** [review]
Apart of the comments, it would be nice to add some example for BTUNE (would require to have `blosc2-btune` installed).

**Comment by FrancescAlted** [comment]
> Apart of the comments, it would be nice to add some example for BTUNE (would require to have `blosc2-btune` installed).

After thinking twice, the best would be to do the example in the `blosc2-btune` project.

**Comment by FrancescAlted** [comment]
> > Apart of the comments, it would be nice to add some example for BTUNE (would require to have `blosc2-btune` installed).
> 
> After thinking twice, the best would be to do the example in the `blosc2-btune` project.

Or perhaps in both, why not .

File: blosc2/core.py (hunk 1 above)
Comment (martaiborra) [reply blosc2/core.py:1235]: Done in 5d6795a2f591109e1b70a1bca2cbeb6c11844cb5

File: blosc2/core.py (hunk 2 above)
Comment (martaiborra) [reply blosc2/core.py]: Fixed in 5d6795a2f591109e1b70a1bca2cbeb6c11844cb5

File: doc/reference/autofiles/top_level/blosc2.Tuner.rst (hunk 3 above)
Comment (martaiborra) [reply doc/reference/autofiles/top_level/blosc2.Tuner.rst:13]: Fixed in 5d6795a2f591109e1b70a1bca2cbeb6c11844cb5

**Review by FrancescAlted — APPROVED** [review]
LGTM

**Pull request merged**